UITests/TransferFunds/TransferUserFundsTest.swift

```diff
@@ -514,9 +517,10 @@ class TransferUserFundsTest: BaseTests {
         XCTAssertTrue(transferFunds.addSelectDestinationSectionLabel.exists)
         XCTAssertTrue(transferFunds.transferAmount.exists)
 
-        let pasteAmountWithCharacters = "1000CAN"
-        transferFunds.pasteAmountToTransferAmount(amount: pasteAmountWithCharacters)
-        XCTAssertEqual(transferFunds.transferAmount.value as? String, "0")
+        // Paste is disabled.
```

Comment:
You can delete this entire test as the entire test if for copy and pate only.

Reply:
Ok sure.